src/domain/signupGroup/utils.ts

```diff
@@ -266,3 +276,31 @@ export const calculateTotalPrice = (
       (priceGroupOptions.find((o) => o.value === curr.priceGroup)?.price ?? 0),
     0
   );
+
+export const getSignupPriceGroupOptions = (
+  registration: RegistrationFieldsFragment,
+  locale: Language
+) => {
+  return (
+    registration.registrationPriceGroups?.map((pg) => {
+      const price = pg?.price ? Number(pg.price) : 0;
+
+      return {
+        label: [
+          `${getLocalisedString(pg?.priceGroup?.description, locale)}`,
+          `${price.toFixed(2).replace('.', ',')} €`,
+        ].join(' '),
+        price,
+        value: pg?.id?.toString() ?? /* istanbul ignore next */ '',
+      };
+    }) ?? /* istanbul ignore next */ []
+  );
+};
+
+export const shouldCreatePayment = (
+  priceGroupOptions: SignupPriceGroupOption[],
+  signups: SignupFormFields[]
+) =>
+  featureFlagUtils.isFeatureEnabled('WEB_STORE_INTEGRATION') &&
+  calculateTotalPrice(priceGroupOptions, signups) > 0 &&
+  signups.every((su) => !su.inWaitingList);
```

Comment:
In the back-end, it is nowadays enough if only one signup is in the attending status in order to be able to create a payment.

Reply:
Changes this so that function returns true if any of the attending signups is not free. I created a separate PR to do the same fix to Linked Registrations UI as well: https://github.com/City-of-Helsinki/linkedregistrations-ui/pull/160